### PR TITLE
Priya kevin edit category label

### PR DESCRIPTION
**Pull request by pspatel04**

### CODE REVIEW

File: .vscode/launch.json [hunk 1]
```diff
@@ -0,0 +1,21 @@
+{
```
Comment (frederickjjoubert) [inline .vscode/launch.json]: I don't think this file should have been committed, could you please add it to the .gitignore

File: src/app/components/popup/popup.component.html [hunk 2]
```diff
@@ -0,0 +1,31 @@
+<mat-card-header>
```
Comment (frederickjjoubert) [inline src/app/components/popup/popup.component.html]: Is this a reusable component or a specific pop up that is only used for the category labels? If you only use it for the category labels it should be renamed categoryLabelPopup or something that makes sense for its purpose.

File: src/proxy.conf.json [hunk 3]
```diff
@@ -1,6 +1,6 @@
 {
   "/api/*": {
-    "target": "http://csse.usc.edu:3000",
```
Comment (frederickjjoubert) [inline src/proxy.conf.json]: @kgrimes2 can you confirm which target should be here?

File: src/app/components/popup/popup.component.html (hunk 2 above)
Comment (pspatel04) [reply src/app/components/popup/popup.component.html]: Its not resuable. I will change the name to categoryLabelPopup.

File: .vscode/launch.json (hunk 1 above)
Comment (pspatel04) [reply .vscode/launch.json]: Yup sure.

**Comment by pspatel04** [comment]
I have made the changes as per requested

**Pull request merged**